src/main/java/com/develhope/spring/vehicle/vehicleEnums/KindOfVehicle.java

```diff
@@ -5,5 +5,17 @@ public enum KindOfVehicle {
     MOTO,
     SCOOTER,
     VAN,
-    TRUCK
+    TRUCK,
+    NOT_DEFINED;
+
+    public static KindOfVehicle convertFromString(String type) {
```

Comment:
Questo al momento effettivamente non serve, ma servirà per il DTO che avevo provato a creare di vehicle per trasformare la stringa in enum. L'ho lasciato per dimenticanza. Sarò sincero se rifaccio un altro push di questo branch vengono purtroppo portate anche le modifche al repack che stanno sull'altro branch e quindi dovrei rifare da capo il branch. Quindi errore mio. In caso non vi andasse bene avere già sto metodo rifarò da capo.

Reply:
Posso aprire un nuovo branch per effettuare le modifiche su codice aggiornato , come in fondo potresti eseguire tu stesso la cosa , dimmi come preferisci fare . 
Ci sentiamo su discord per accordarci.